src/unconventional-superconductivity.md

```diff
@@ -37,7 +43,16 @@ Conventional superconductors are characterized by the global U(1) phase rotation
 - <time data-format="H:mm" datetime="2022-03-31T15:30:00+02:00">15:30</time> Aline Ramires, Paul Scherrer Institut
 - <time data-format="H:mm" datetime="2022-03-31T16:10:00+02:00">16:10</time> Andrey Chubukov, University of Minnesota
 - <time data-format="H:mm" datetime="2022-03-31T16:50:00+02:00">16:50</time> break
-- <time data-format="H:mm" datetime="2022-03-31T17:00:00+02:00">17:00</time> Young Researchers' session
+- <time data-format="H:mm" datetime="2022-03-31T17:00:00+02:00">17:00</time> Satoshi Ikegaya, Nagoya University,  
```

Comment:
Are you sure this is OK? That's 1:00 in Japan...

Reply:
They applied, and I placed them as early as possible... So really, no other options. Actually there were 2 people from that group applying, so I am sure they are aware of it.

Reply:
I personally had experience of participating in the US conference until past midnight and in the Australian workshop starting at 5-6 am, so I guess, it depends on motivation :-)